docs/queries.rst

```diff
@@ -57,24 +61,35 @@ Show **only** these fields on the model:
     class QuestionType(DjangoObjectType):
         class Meta:
             model = Question
-            only_fields = ('question_text')
+            fields = ('id', 'question_text')
+
+You can also set the ``fields`` attribute to the special value ``'__all__'`` to indicate that all fields in the model should be used.
 
+For example:
 
-exclude_fields
-~~~~~~~~~~~~~~
+.. code:: python
 
-Show all fields **except** those in ``exclude_fields``:
+    class QuestionType(DjangoObjectType):
+        class Meta:
+            model = Question
+            fields = '__all__'
+
+
+``exclude``
+~~~~~~~~~~~
+
+Show all fields **except** those in ``exclude``:
 
 .. code:: python
 
     class QuestionType(DjangoObjectType):
         class Meta:
             model = Question
-            exclude_fields = ('question_text')
+            exclude = ('question_text')
```

Comment:
should be here tuple?
```suggestion
            exclude = ('question_text', )
```

Reply:
Yes you're right it should be. Will fix the other examples as well. Thanks!